coverage.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
can this be moved to a scripts/ directory? I think we should avoid littering the root directory with a bunch of utility scripts.

I'm also putting stuff into a scripts directory: https://github.com/open-telemetry/opentelemetry-python/pull/228/files

Reply:
It's done. Thank you!